src/main/js/controls/Line/helpers/helpers.js

```diff
@@ -17,7 +17,8 @@ import errors from "../../../helpers/errors";
 import {
     legendClickHandler,
     legendHoverHandler,
-    loadLegendItem
+    loadLegendItem,
+    legendSelected
```

Comment:
Can you make the function name as `isLegendSelected`.
Any function that returns a boolean needs to have that convention. https://stackoverflow.com/a/1370847/5727925

Reply:
updated here https://github.com/cerner/carbon-graphs/pull/140/commits/dba15f55a12996a5f56a0ef4bcaec4f66e1b09f1.